### DISCUSSION TITLE
Quasar v2 in Tesla in-car web browser

### DISCUSSION THREAD

**Discussion by pikil** [quasarframework/quasar#11656] [Show and Tell]
Hi All,
It's not something extraordinary for an in car web browser, of course, but in case someone was wondering if Quasar Framework works in Tesla browser - it does work like a charm. No problems/misalignments/non-responsiveness detected.
In case, someone wants to check a specific component/plugin (button, dropdown, etc.), you can leave a comment with the quasar.dev link here.

The websites tested so far:
- https://quasar.dev
   - _Quasar V2_;&emsp;:white_check_mark:
   - _Dialog plugin_;&emsp;:white_check_mark:
   - _Select component_;&emsp;:white_check_mark:

![IMG_3693](https://user-images.githubusercontent.com/26779040/145730658-38c9711f-7335-4a45-8c01-018718f0f79f.JPG)

![IMG_3692](https://user-images.githubusercontent.com/26779040/145730661-eb1efa11-ee60-42e9-bd84-b24f38ae5ce0.JPG)

- https://app.coloban.com
   - _Quasar V2;_&emsp;:white_check_mark:
   - _Quasar Dark plugin;_&emsp;:white_check_mark:&emsp;(in both Manual and Auto modes)
   - _Websockets;_&emsp;:white_check_mark:
   - _chart.js;_&emsp;:white_check_mark:
   - _@quasar/quasar-ui-qcalendar;_&emsp;:white_check_mark:
   - _dhtmlx-gantt;_&emsp;(works almost fine, drag elements are not active, visual representation as usual*)
   - _dexie (IndexedDB);_&emsp;:white_check_mark:
   - _SortableJS;_&emsp;:white_check_mark:
   - _HammerJS;_&emsp;:white_check_mark:
   - _WebRTC;_&emsp;(*not working*)

![IMG_3688](https://user-images.githubusercontent.com/26779040/145730669-b096ea4c-bb81-48a7-82fe-b45ba5ba1c39.JPG)

![IMG_3690](https://user-images.githubusercontent.com/26779040/145730674-48a4d8c9-fd85-4a18-bec1-dea438017f9b.JPG)

**Comment by pdanpdan** [comment]
According to [this](https://webhostingbuddy.com/blog/5-tips-to-optimize-your-website-for-teslas-in-car-web-browser-model-s-3-x-y/) the browser should be chromium, so it should all work.

**Comment by pikil** [comment]
That seems to be correct,
but it doesn't mean that that in-car browser implements all the Chromium features in full.

For example, WebRTC is not operating and there is something happening with gantt-html (non-responsive), but I cannot figure out what exactly for now...

**Comment by pikil** [comment]
No Quasar-related issues spotted, though :tada:

**Comment by rstoenescu** [comment]
So cool seeing Quasar being used in all sorts of places :)
Thanks for sharing!

**Comment by hawkeye64** [comment]
@pikil Great to see QCalendar being used there. As Razvan said, thank you for sharing. :heart:
